### ISSUE TITLE
reproduce the performance of YOLOV8s,YOLOV9s,YOLOV10s on COCO.

### ISSUE THREAD

**Issue by Zhengcv** [ultralytics/ultralytics#18150]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Has anyone reproduced the performance of YOLOv8, YOLOv9, and YOLOv10 on the COCO dataset based on the official configuration files? The results I reproduced do not match the official reported performance. I would like to know the typical performance range for others who have attempted the reproduction.

The training parameters for YOLOv8 only use scale augmentation in the range of 0.5 to 1.5 and mosaic augmentation, correct? It doesn't use mixup or copy-paste data augmentation? When printing the corresponding weights, it shows only the aforementioned methods. However, I am concerned about the issue of training parameters being overwritten due to the close-mosaic setting in previous YOLO versions.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ZZxiaoxiao, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report or concerns reproducibility issues, we encourage you to provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and debug the problem. This could include your training script, specific parameters used, and dataset details.

If you are seeking ❓ clarification on training settings or augmentation methods, it would be helpful to share the exact YAML configuration file you are using, along with relevant training logs or outputs. Please also verify that you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

For exploring tasks like COCO benchmark performance and settings, upgrading to the latest `ultralytics` version is often beneficial to ensure you are using the most recent updates. You can do this with:

```bash
pip install -U ultralytics
```

YOLO models operate seamlessly in verified environments. You can run them effortlessly in any of the following:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

The community is full of experts and enthusiasts who can assist you in your exploration of the COCO reproduction tasks! For real-time discussions, join us on [Discord](https://discord.com/invite/ultralytics) 🎧. For more in-depth threads, participate in discussions on our [Discourse forums](https://community.ultralytics.com) or [Subreddit](https://reddit.com/r/Ultralytics).

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response to help streamline support 🛠️. An Ultralytics engineer will review your question and provide further assistance soon. 😊

**Comment by Y-T-G** [comment]
You can get the args used from the model.

https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030

**Comment by ZZxiaoxiao** [comment]
model parameters for object detection

**Comment by glenn-jocher** [comment]
For object detection, YOLO models like YOLOv8, YOLOv9, and YOLOv10 use specific training parameters, which include learning rate, batch size, image size, and augmentations like Mosaic and Scale. You can view and modify these parameters in the YAML configuration files or through the `model.train` function. For details on these models and their configurations, refer to the [Ultralytics YOLO Docs](https://docs.ultralytics.com/).
